images/base/files/usr/local/bin/entrypoint

```diff
@@ -18,6 +18,23 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+configure_containerd() {
+  # we need to switch to the 'native' snapshotter on zfs
+  if [[ "$(stat -f -c %T /kind)" == 'zfs' ]]; then
```

Comment:
do we still need this https://github.com/kubernetes-sigs/kind/blob/master/pkg/cluster/internal/providers/docker/provision.go#L208 ?

Reply:
yes

Reply:
do we need to handle btrfs too?

Reply:
we have prior art for zfs's best option being falling back to the native driver because the zfs mode is sketchy at least in the kind context and overlay driver doesn't work.
we don't for btrfs.

if someone reports this with evidence, we can add it. I _think_ btrfs is already fixed but can't confirm.

Reply:
see: https://github.com/kubernetes-sigs/kind/issues/1719